battleship.py

```diff
@@ -1,4 +1,17 @@
 # Functions
+def retrieve_size():
+    difficulty = input("Enter difficulty: 1, 2, or 3.")
+    if difficulty == "1":
```

Comment:
this should work for now but it's best to use if elif instead of seperate if statements. 
like 
```
   if difficulty == "1":
        return 16
    elif difficulty == "2":
        return 32
    elif difficulty == "3":
        return 64
    else:
        print("Invalid difficulty! Try again.")
        retrieve_size()
```
Because if you use seperate ifs it will check all of them regardless if the first one was true. It doesn't make much of a difference in this case but it's bad practice.